spyder/plugins/findinfiles.py

```diff
@@ -52,11 +52,13 @@ def __init__(self, parent=None):
         exclude_regexp = self.get_option('exclude_regexp')
         in_python_path = self.get_option('in_python_path')
         more_options = self.get_option('more_options')
+        case_sensitive = self.get_option('case_sensitive', True)

```

Comment:
Please don't use a default value here. Instead, add `case_sensitive` as an option of the `find_in_files` section in `config/main.py`.